src/transform/plugins/includes/collect.ts

```diff
@@ -23,30 +24,34 @@ function processRecursive(
     };
 
     try {
-        const content = copyFile(includePath, targetDestPath, includeOptions);
+        const contentProcessed = copyFile(includePath, targetDestPath, includeOptions);
 
         // To reduce file reading we can include the file content into the generated content
-        if (included && content) {
-            const includedRelativePath = getRelativePath(includedParentPath, includePath);
-
-            // The appendix is the map that protects from multiple include files
-            if (!appendix.has(includedRelativePath)) {
-                // Recursive function to include the depth structure
-                const includeContent = collectRecursive(
-                    content,
-                    {
-                        ...options,
-                        path: includePath,
-                        includedParentPath,
-                    },
-                    appendix,
-                );
-
-                // Add to appendix set structure
-                appendix.set(
-                    includedRelativePath,
-                    `{% included (${includedRelativePath}) %}\n${includeContent}\n{% endincluded %}`,
-                );
+        if (included) {
+            const content = contentProcessed ?? readFileSync(targetDestPath, 'utf8');
```

Comment:
`copyFile` do not return anything 
https://github.com/diplodoc-platform/cli/blob/9ad49e878b02cb742dccd8129ee19294d9e35253/src/resolvers/md2md.ts#L80

why not just `const content = readFileSync(targetDestPath, 'utf8');` ?

Reply:
This is implemented for testing purposes